examples/tensorflow/image_recognition/keras_models/resnetv2_101/quantization/ptq/main.py

```diff
@@ -59,70 +55,73 @@
-          predictions, labels = postprocess((predictions, labels))
-          metric.update(predictions, labels)
-      return results
-
-  results = eval_func(eval_dataloader, metric)
-  acc = metric.result()
-  return acc
+    """Custom evaluate function to inference the model for specified metric on validation dataset.
+
+    Args:
+        model (tf.saved_model.load): The input model will be the class of tf.saved_model.load(quantized_model_path).
+        measurer (object, optional): for benchmark measurement of duration.
+        
+    Returns:
+        accuracy (float): evaluation result, the larger is better.
+    """
+    infer = model.signatures["serving_default"]
+    output_dict_keys = infer.structured_outputs.keys()
+    output_name = list(output_dict_keys )[0]
+    postprocess = LabelShift(label_shift=1)
+    metric = TensorflowTopK(k=1)
+
+    def eval_func(dataloader, metric):
+        results = []
+        for idx, (inputs, labels) in enumerate(dataloader):
+            inputs = np.array(inputs)
+            input_tensor = tf.constant(inputs)
+            if measurer:
+                measurer.start()
+            predictions = infer(input_tensor)[output_name]
+            if measurer:
+                measurer.end()
+            predictions = predictions.numpy()
+            predictions, labels = postprocess((predictions, labels))
+            metric.update(predictions, labels)
+        return results
+
+    results = eval_func(eval_dataloader, metric)
+    acc = metric.result()
+    return acc
 
 def main(_):
-  if FLAGS.tune:
-    from neural_compressor.experimental import Quantization, common
-    quantizer = Quantization(FLAGS.config)
-    quantizer.model = common.Model(FLAGS.input_model)
-    quantizer.eval_func = evaluate
-    quantizer.calib_dataloader = calib_dataloader
-    q_model = quantizer.fit()
-    q_model.save(FLAGS.output_model)
-
-
-  if FLAGS.benchmark:
-    from neural_compressor.experimental import Benchmark, common
-    evaluator = Benchmark(FLAGS.config)
-    evaluator.model = common.Model(FLAGS.input_model)
-    evaluator.b_func = evaluate
-    evaluator.b_dataloader = eval_dataloader
-    evaluator(FLAGS.mode)
+    if FLAGS.tune:
+        from neural_compressor import quantization
+        from neural_compressor.config import PostTrainingQuantConfig
+
+        conf = PostTrainingQuantConfig(calibration_sampling_size=[50, 100])
+        q_model = quantization.fit(FLAGS.input_model, conf=conf, calib_dataloader=calib_dataloader,
+                    eval_func=evaluate)
+        q_model.save(FLAGS.output_model)
+
+    if FLAGS.benchmark:
+        from neural_compressor import benchmark
+        from neural_compressor.config import BenchmarkConfig
+        if FLAGS.mode == 'performance':
+            conf = BenchmarkConfig(iteration=100, cores_per_instance=4, num_of_instance=7)
+            benchmark.fit(FLAGS.input_model, conf, b_dataloader=eval_dataloader, b_func=evaluate)
```

Comment:
b_func should be removed, otherwise the iteration will not be controlled by INC inside.